test/evergreen.yml

```diff
@@ -1811,10 +1862,12 @@ tasks:
         vars:
           test_config: debug_mode=(cursor_copy=true)
           test_config_filename: configs/bounded_cursor_stress_reverse_default.txt
-          test_name: bounded_cursor_stress
+          test_name: bounded_cursor_stress_reverse
 
+  # Also in combined test cppsuite-combined-cursor-copy.
```

Comment:
Can you remove all the tasks that are disabled?

Reply:
Done